modules/dynamic-plugins/con-dynamic-plugins-cache.adoc

```diff
@@ -0,0 +1,66 @@
+[id="con-dynamic-plugin-cache_{context}"]
+
+= Using the dynamic plugins cache
+The dynamic plugins cache feature in {product} ({product-very-short}) optimizes the installation process of dynamic plugins and reduces platform boot time. This feature caches previously installed plugins, eliminating the need to re-download them on subsequent boots if their configuration has not changed.
+
+When you enable dynamic plugins cache, the feature works as follows:
+
+* The system calculates a checksum of each plugin's YAML configuration (excluding `pluginConfig`).
+* This checksum is stored in a file named `dynamic-plugin-config.hash` within the plugin's directory.
+* During boot, if a plugin's package reference matches the previous installation and the checksum is unchanged, the download is skipped.
+* Plugins that have been disabled since the previous boot are automatically removed.
+
+== Enabling the dynamic plugins cache
```

Comment:
I suggest applying the mod-docs procedural module template. It would be helpful to include prerequisites and a procedure tag with an ordered list of steps.